website/content/ja/contribute/projects/platform-maturity-model-examples/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: プラットフォーム成熟度モデル例
```

Comment:
English comments following:

タイトルだけを翻訳した英語版のコピーは、作らない方がいいかもしれません。

英語版を手動でコピーしてコミットしていたのは以前の名残でして、現在では、未翻訳の各国語のコンテンツはサイトが公開されるときに自動的に英語からコピーされます。

タイトルやサイドメニューの項目が翻訳されるというメリットはありますが、英語版に更新があったときの追従のコストに比べると、メリットが少ない気がします。

---

Here is the translation:

---

It might be best not to create a copy of the English version with only the titles translated.

Manually copying and committing the English version was a remnant of past practices. Nowadays, untranslated content in various languages is automatically copied from English when the site is published.

While there is the advantage of having titles and sidebar menu items translated, the benefit seems minimal compared to the cost of keeping up with updates to the English version.